### ISSUE TITLE
Release 6.7.0 links to a commit in develop branch; develop branch and main branch have differents commits until release 6.7.0

### ISSUE THREAD

**Issue by HectorNM** [coralproject/talk#3398]
Hi everyone, I was updating to release 6.7.0 and I noticed something strange here in Github project:
- Release 6.7.0 points to commit `3e4f91af30519c8af6bb3660be8ee385626952b9`, which is on branch `develop`. Is that correct? Am I checking this correctly?
- `main` branch has a commit called "Release 6.6.2", which in branch `develop` is broken into two commits. Which one is the correct one?

Could you take a look into this? Or is this correct? Should I use commits from `develop` branch? Sorry if I misunderstood something.

Thank you!

**Expected behavior:** ---------

**Actual behavior:** ---------

**Related Issues:**  ---------

**Versions:** ---------

- NodeJS:
- NPM:
- MongoDB:
- Redis:
- Browser:
- OS:

**Comment by wyattjoh** [comment]
Hi @HectorNM! We discovered this in parallel. We're releasing 6.7.1 that will address this and fixes another issue.

Let me know if that fixes that issue!
